main_test.go

```diff
@@ -54,14 +54,21 @@ func (t *MainTest) TestGetUserAgentWhenMetadataImageTypeEnvVarIsSet() {
 	defer os.Unsetenv("GCSFUSE_METADATA_IMAGE_TYPE")
 
 	userAgent := getUserAgent("AppName")
-	expectedUserAgent := strings.TrimSpace(fmt.Sprintf("gcsfuse/%s %s %s", getVersion(), "AppName", os.Getenv("GCSFUSE_METADATA_IMAGE_TYPE")))
+	expectedUserAgent := strings.TrimSpace(fmt.Sprintf("gcsfuse/%s AppName (GPN:gcsfuse-DLVM)", getVersion()))
```

Comment:
I think you should use os.Getenv("GCSFUSE_METADATA_IMAGE_TYPE") instead of "DLVM" directly as we are testing environment variable also.

Reply:
Please correct me if I am wrong, but, os.Getenv() is being used in the getUserAgent() method. Using os.Getenv here won't catch any failures if something goes wrong in the os.Getenv method as if it will have the wrong value in both expected and tested user agent.